### PR TITLE
Update cli.md: add dbus access for Raspberry

### PR DESCRIPTION
**Pull request by tbertels**
## Proposed change

Add "-v /run/dbus:/run/dbus:ro" (without quotes) to the Raspberry container configuration to fix Bluetooth issue.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes [#76151](https://github.com/home-assistant/core/issues/76151)

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by agners — APPROVED** [review]

**Review by agners — DISMISSED** [review]
Actually, on second thought, this is already documented in the Bluetooth part:

https://www.home-assistant.io/integrations/bluetooth/#additional-details-for-container-core-and-supervised-installs

Exposing the D-Bus System Bus to Home Assistant Core could also be seen as a security risk. Users of Container installation are expected to manage their system on their own, and find the required settings useful for their installation. I don't think we should recommend to use this by default.

If anything, this should be a separate section like "Exposing devices" right now. We could also just add a hint and link to the Bluetooth documentation.

**Comment by tbertels** [comment]
There doesn't seem to be any warning about a potential security risk at https://www.home-assistant.io/integrations/bluetooth/#additional-details-for-container-core-and-supervised-installs

Currently, the user only gets this error message:
``Config entry 'DC:A6:32:*' for bluetooth integration not ready yet: hci0 (DC:A6:32:*): DBus service not found; docker config may be missing `-v /run/dbus:/run/dbus:ro`: {ex}; Retrying in background``
So it doesn't seem worth it to make the user jump through an additional hoop.

I would suggest either disabling the Bluetooth integration by default for the docker version of Home Assistant or adding D-Bus access in the configuration while warning the user about the potential security risk.
But I don't think there should be an error message with the standard configuration.

**Comment by agners** [comment]
Hm, how do we detect Bluetooth devices in first place? Not through D-Bus then I guess? :thinking: 

@bdraco thoughts?

**Comment by bdraco** [comment]
It should not be able to detect new ones if dbus is disabled 

We do get a significant amount of support in the Bluetooth queue, discord, and other channels since the dbus mount is missing in the docs.

The Dbus mount is required if you are using Bluetooth. An alternative to giving the Home Assistant container access to Dbus is to use an [ESPHome Bluetooth proxy](https://esphome.github.io/bluetooth-proxies/).

**Comment by frenck** [comment]
@agners @bdraco How do we continue?

Alternatively, we could also add a note to the installation docs referring to Bluetooth (and not directly add it by default, but still have it there).

../Frenck

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @tbertels :+1:

**Comment by c0ffeeca7** [comment]
> @agners @bdraco How do we continue?
> 
> Alternatively, we could also add a note to the installation docs referring to Bluetooth (and not directly add it by default, but still have it there).
> 
> ../Frenck

feedback from @jesserockz : 

> It can be merged. There will be a separate pr for the code comment changes

File: source/_includes/installation/container/cli.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
       --privileged \
       --restart=unless-stopped \
       -e TZ=MY_TIME_ZONE \
-      -v /PATH_TO_YOUR_CONFIG:/config \
+      -v /PATH_TO_YOUR_CONFIG:/config -v /run/dbus:/run/dbus:ro \
```
Comment (agners) [inline source/_includes/installation/container/cli.md]:
```suggestion
      -v /PATH_TO_YOUR_CONFIG:/config \
      -v /run/dbus:/run/dbus:ro \
```

File: source/_includes/installation/container/cli.md [hunk 2]
```diff
@@ -39,7 +39,7 @@
       --restart=unless-stopped \
       --privileged \
       -e TZ=MY_TIME_ZONE \
-      -v /PATH_TO_YOUR_CONFIG:/config \
+      -v /PATH_TO_YOUR_CONFIG:/config -v /run/dbus:/run/dbus:ro \
```
Comment (agners) [inline source/_includes/installation/container/cli.md]:
```suggestion
      -v /PATH_TO_YOUR_CONFIG:/config \
      -v /run/dbus:/run/dbus:ro \
```

**Review by agners — COMMENTED** [review]
I like the idea mentioned by Frenck. I suggest to also add a note at the top saying that D-Bus is required for Bluetooth and linking to the integration.

**Review by frenck — APPROVED** [review]
Thanks, @tbertels & @agners 👍 

../Frenck

**Pull request merged**